{% extends "layout.html" %}
  
{% block title %}Design{% endblock %}
{% block lead %}Hardware in iGEM should make synthetic biology based on standard parts easier, faster, better, or more accessible to our community.{% endblock %}

{% block extraStyles %}
<link href="{{ url_for('static', filename = 'css/design.css') }}" rel="stylesheet">

{% endblock %}

{% block page_content %}
<div class="standard-header banner-v1 curvy-divider">
    <h1> Design </h1>
    <div class="simple-wave-divider">
        <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
            <path d="M985.66,92.83C906.67,72,823.78,31,743.84,14.19c-82.26-17.34-168.06-16.33-250.45.39-57.84,11.73-114,31.07-172,41.86A600.21,600.21,0,0,1,0,27.35V120H1200V95.8C1132.19,118.92,1055.71,111.31,985.66,92.83Z" class="shape-fill"></path>
        </svg>
    </div>
</div>

<div class="d-flex flex-column justify-content-center align-items-center">
    <div class="body-container wave-div narrow">
        <div class="container">
          <div class="about-section d-flex flex-row justify-content-center align-items-center">
            <div class="wrapper">
              <h2>What is an LFA?</h2>
              <p>Lateral flow assays (LFAs) are paper-based detection methods. Lateral flow immunoassays, a specific type of LFA, use a series of antibodies to detect whether a biomarker is present in a liquid sample. (Lateral flow assays - PMC (nih.gov))</p>
            </div>
            <div class="img-wrapper bunch-of-lfas">
              <img style="max-height: 15rem; padding: 2rem;"src="https://static.igem.wiki/teams/4477/wiki/images/lfa-group.svg" alt="lfa bunch">
            </div>
          </div>
        </div>
    </div>
</div>

<div class="d-flex flex-column justify-content center">
    <div class="d-flex flex-row justify-content-center body-padding">
        <div class="index-padding">
            <div class="fixed-index">
                <p>FIXED INDEX will go here once styled</p>
            </div>
        </div>
       
    
        <div class="body-container d-flex flex-column align-items-center justify-content-center" data-aos="fade-up" data-aos-duration="1000" data-aos-easing="ease-out-sine">
            <div class="container">
                <div class="body-section top-page">
                    <div class="lfa-beginning">
                        <div class="text-wrapper">
                            <h2> How does an LFA Work? </h2>
                        </div>
                        <div class="image-wrapper d-flex flex-column justify-content-center align-items-center lfa-diagram img-fluid">
                            <img src="https://static.igem.wiki/teams/4477/wiki/images/diag-lfadiagram.svg" alt="lfa diagram">

                            <div class="subtitle-wrapper pt-4">
                                <p>Diagram of an LFA. Most LFAs include a sample pad, conjugate pad, absorbent pad, nitrocellulose membrane, and absorbent pad, along with the test and control lines to detect a biomarker. </p>
                            </div>
                        </div>
                        
                    </div>
                        <div class="diagram">
                            <div class="steps-container">
                                <div class="text-wrapper-des">
                                    <p>
                                        First, a <span class="bold-text">sample is applied onto the sample pad</span>, which is designed to prepare the sample for optimum flow through the rest of the test strip.
                                    </p>
                                </div>
                                <img src="https://static.igem.wiki/teams/4477/wiki/images/diag-samplepad.svg" alt="sample applied">
                            </div>

                            <div class="steps-container">
                                <img src="https://static.igem.wiki/teams/4477/wiki/images/diag-conjugatepad.svg" alt="">

                                <div class="text-wrapper-des">
                                    <p>
                                        The sample flows through to the <span class="bold-text">conjugate release pad</span> that then is absorbed by the test membrane. <span class="bold-text">Antibodies</span> on the conjugate pad <span class="bold-text">bind to the biomarker</span> in the sample. 
                                    </p>
                                </div>
                            </div>

                            
                            <div class="steps-container">
                                <div class="text-wrapper-des">
                                    <p>
                                        This <span class="bold-text">primary antibody</span> is labeled with <span class="bold-text">nanoparticles</span> that allow these antibodies to be detected with the <span class="bold-text">naked eye.</span> 
                                        If the <span class="bold-text">molecule of interest</span>, such as the <span class="bold-text">biomarker</span>, is in the sample, these labeled primary antibodies <span class="bold-text">bind</span> to the molecule and lead to the <span class="bold-text">visualization of colored lines</span> later in the test.
                                    </p>
                                </div>
                                <img src="https://static.igem.wiki/teams/4477/wiki/images/diag-igggold.svg" alt="">
                            </div>
                            

                            <div class="steps-container">
                                <img style="max-height: 23rem;" src="https://static.igem.wiki/teams/4477/wiki/images/diag-samplefront.svg" alt="nitrocellulose pad">
                                <div class="text-wrapper-des">
                                    <p>The target molecule/labeled antibody complexes travel onto a <span class="bold-text">nitrocellulose membrane</span>, where <span class="bold-text">two more antibodies</span> are immobilized in lines.</p>
                                </div>
                            </div>

                            <div class="steps-container-thicc">
                                <div class="text-wrapper-des">
                                    The complexes reach the <span class="bold-text">test line</span> first, where another <span class="bold-text">primary antibody</span> targeting the <span class="bold-text">molecule of interest</span> is situated.
                                </div>
                            </div>

                            <div class="steps-container-thicc">
                                <div class="text-wrapper-des">
                                    <p>
                                        The target molecule binds this other primary antibody, and the whole complex becomes <span class="bold-text">immobilized</span> at the test line, forming a <span class="bold-text">“sandwich complex”</span>.
                                        This other primary antibody is distinct from the first primary antibody and recognizes a <span class="bold-text">different epitope</span> on the molecule of interest. The labeled antibodies show up as a <span class="bold-text">colored line</span>, signaling detection of the target molecule.
                                    </p>
                                </div>

                                <img src="https://static.igem.wiki/teams/4477/wiki/images/diag-samcomp.svg" alt="sandwich complex">
                                

                            </div>

                            <div class="steps-container-thicc">
                                <div class="text-wrapper-des">
                                    <p>The labeled antibody is <span class="bold-text">unable to bind</span> to the test line if the biomarker is <span class="bold-text">not present</span> in the sample. Regardless of whether the target molecule is in the sample, the labeled antibody continues down the nitrocellulose membrane and <span class="bold-text">reaches the control line.</span></p>
                                </div>
                            </div>

                            <div class="steps-container-thicc">
                                <div class="text-wrapper-des">
                                    <p>Here, a <span class="bold-text">secondary antibody</span> that binds to the labeled antibody itself is immobilized. If the test is working properly, the <span class="bold-text">secondary and labeled antibody will bind</span>, causing the control line to show a <span class="bold-text">colored line</span>. The sample continues off the membrane and is <span class="bold-text">absorbed</span> by an absorbent pad, <span class="bold-text">stopping the fluid flow</span>.   Lateral flow assays - PMC (nih.gov)</p>
                                </div>
                            </div>

                            <div class="img-wrapper large">
                                <img src="https://static.igem.wiki/teams/4477/wiki/images/diag-posneg.svg" alt="positive and negative results">
                            </div>
                        </div>
                    
                </div>
            </div>

            <div class="container">
                <div class="body-section">
                    <h2>How Our LFA is Unique</h2>
                    <div class="text-wrapper">
                        <p>
                            For our LFA, we decided to test for the presence of oxidized low density lipoprotein (oxLDL) because oxLDL is a biomarker of atherosclerosis found in the blood (The changing landscape of atherosclerosis | Nature). Then, if our test detected oxLDL, the patient would have evidence they are experiencing early atherosclerosis. 

                            Our strip contains one full-length antibody, an scFv fragment, and a VHH nanobody fragment Nanobody-based products as research and diagnostic tools | Elsevier Enhanced Reader, all which are produced in <span class="italic-text">E. coli</span> SHuffle. To detect oxLDL, we designed two primary antibodies. The labeled primary antibody binds to a protein component of oxLDL called MDA-modified apolipoprotein B (apoBMDA) Recombinant Human Antibodies Against Aldehyde-Modified Apolipoprotein B-100 Peptide Sequences Inhibit Atherosclerosis | Circulation (ahajournals.org), while the other primary antibody immobilized at the test line binds to oxidized phospholipids within oxLDL. The fact that these two primary antibodies bound to different epitopes helped ensure the antibodies did not interfere with the other’s binding site. Lastly, we designed a secondary antibody targeting the constant domain of the labeled primary antibody and used this on the control line to verify the test worked correctly.
                        </p>
                    </div>

                    <h4>What is Unique to our Design</h4>
                    <ul>
                        <li>Utilizes a special strain of <span class="italic-text">E. coli</span> to produce antibodies</li>
                        <li>LFA incorporates non-full length antibodies</li>
                        <ul>
                            <li>These antibody fragments are equally capable of recognizing epitopes just as full-length antibodies are, but are less complex and easier to develop (A gold nanoparticle-single-chain fragment variable antibody as an immunoprobe for rapid detection of morphine by dipstick - RSC Advances (RSC Publishing) DOI:10.1039/C7RA12810J)</li>
                        </ul>
                        <li>Full-length antibody is a recombinant of human and mouse genes</li>
                    </ul>
                </div>
            </div>

            <div class="container">
                <div class="body-section">
                    <h2>Choosing a Chassis</h2>
                    <div class="text-wrapper">
                        <p>
                            An LFA requires the production of three different antibodies, which have complex structures containing disulfide bonds. Most bacteria, like <span class="italic-text">E. coli</span>, can only produce the disulfide bonds necessary for antibody production in their periplasm, because their cytoplasm is a reducing environment. This is not favorable for the formation of disulfide bonds. In order for the bonds to be formed, modification of the cytoplasm is vital for utilizing the cytoplasmic space to produce proteins. One group of scientists developed a strain of <span class="italic-text">E. coli</span> with an oxidizing cytoplasm that allows the formation of disulfide bonds, and this strain is SHuffle Efficient expression of full-length antibodies in the cytoplasm of engineered bacteria (nature.com). We determined through modeling that all of our antibodies and antibody fragments have disulfide bonds and the number of cysteines fall within the optimal range of our choice, so we chose the B-strain of <span class="italic-text">E. coli</span> SHuffle.  
        
                        </p>
                    </div>
                </div>
            </div>

            <div class="container">
                <div class="body-section">
                    <h4>How SHuffle Works</h4>
                    <div class="text-wrapper">
                        <p>
                            SHuffle differs from standard <span class="italic-text">E. coli</span> in that it maintains a more oxidizing cytoplasm, which allows cells to correctly form disulfide bonds in proteins, which enables researchers to overexpress correctly folded proteins. (https://microbialcellfactories.biomedcentral.com/articles/10.1186/1475-2859-11-56) 
                            Several major genetic modifications result in this modified cytoplasm. Two genes coding for cytoplasmic reductases, specifically thioredoxin reductase (trxB) and glutathione reductase (gor), are knocked out from the <span class="italic-text">E. coli</span> genome. While this leads to a less reducing and therefore more oxidizing cytoplasm, these two mutations on their own are lethal to the <span class="italic-text">E. coli</span>. This is due to a lack of reductases recycling other essential enzymes into their reduced state. So, another mutation called AhpC* is introduced into the genome, modifying 2-Cys alkyl hydroperoxide reductase. This mutation returns reducing power to an enzyme called glutaredoxin 1 (Grx1) and makes the previously described double knock out non-lethal to <span class="italic-text">E. coli</span>. In the oxidizing environment, thioredoxin enzymes including thioredoxin 1 (Trx1) are able to form disulfide bonds between cysteine residues of proteins. However, these thioredoxins form disulfide bonds indiscriminately between any two cysteine residues, meaning that the protein could be misfolded at this stage. So, researchers overexpress an enzyme called disulfide bond isomerase c (DsbC), which is naturally found in the periplasm of <span class="italic-text">E. coli</span>. This enzyme moves, or shuffles, mis-formed disulfide bonds to the correct conformations which finally results in correctly folded proteins in the cytoplasm.
                        </p>
                    </div>
                </div>
            </div>

            <div class="container">
                <div class="body-section">
                    <h2>Plasmid Design</h2>
                    <div class="text-wrapper">
                        <p>
                            For each antibody, we designed a composite insert. Each insert was complete with a strong T7 promoter, an <span class="italic-text">E. coli</span> codon-optimized ribosomal binding site (RBS), the antibody coding sequence, and a terminator. Between each component, we included non-illegal restriction sites for modular design. This is so that other individuals or iGEM teams can swap out parts if needed to vary the rate of transcription or coding sequence. This is also very essential in transformation. 

                            <br>

                            We decided to biobrick a composite part for each antibody in order to use standard assembly. Having a single synthesized component only requires one step to assemble our plasmid. The backbone we used was pSB3K3, a low-copy number plasmid. Our goal was to synthesize functional antibodies, so we chose a low-copy plasmid to limit the metabolic burden on our cells, increasing the success of our antibodies being synthesized correctly.

                        </p>
                    </div>
                </div>
            </div>
        

            <div class="container">
                <div class="body-section">
                    <h2>Designing Our Antibodies</h2>
                    
                    <div class="col-md">
                        <img class="abimg" src="https://static.igem.wiki/teams/4477/wiki/fullab.svg" />
                    </div>
                    <div class="col-md">
                        <h4>Labelled Anti-ApoB<sub>MDA</sub> Design</h4>
                    <p>
                        We decided to develop a full-length IgG to use as our labeled anti-apoB(MDA) primary antibody. While antibody fragments that omit the constant domain of IgGs are simpler to produce, we chose to include the entire IgG for this antibody because the antibody needed its constant domain to bind properly to the gold nanoparticles we used to label it. 
                        The sequence of the variable domain of this antibody was based on that of Orticumab, a human anti-apoB(MDA) antibody currently being developed Recombinant Human Antibodies Against Aldehyde-Modified Apolipoprotein B-100 Peptide Sequences Inhibit Atherosclerosis | Circulation (ahajournals.org. The constant domain of the antibody was identical to the constant domain of a mouse IgG1. We chose to substitute the mouse constant domain in place of the human constant domain of Orticumab because there are many secondary antibodies that bind to our chosen mouse constant domain A toolbox of anti–mouse and anti–rabbit IgG secondary nanobodies | Journal of Cell Biology | Rockefeller University Press (rupress.org) , which made our design for the secondary antibody easier.
                    </p>
                    </div>
                </div>
            </div>

            <div class="container">
                <div class="body-section">
                    <h4>Anti-OxLDL Design</h4>

                    <div class="text-wrapper">
                        <p>
                            We also designed two other primary antibodies that could be used to bind to the test line. These were inspired by anti-oxLDL antibodies we found in literature, called IK-17 (Human-Derived Anti–Oxidized LDL Autoantibody Blocks
                            Uptake of Oxidized LDL by Macrophages and Localizes to Atherosclerotic Lesions In Vivo) and MCPC603 (The Three-Dimensional Structure of a Phosphorylcholine-Binding Mouse Immunoglobulin Fab and the Nature of the Antigen Binding Site | PNAS). Like our labeled antibody, IK-17 binds to the apoB protein that encircles oxLDL, but it binds to a different epitope than our labeled antibody (we used protein modeling to ensure this and verify that IK-17 and the labeled antibody wouldn't prevent the other from binding). MCPC603, however, binds to oxidized phospholipids within the lipid core of oxLDL. One of our criteria in our literature search for these other anti-oxLDL antibodies was for them to be antibody fragments as opposed to full-length antibodies. Our rationale for this choice was that antibody fragments are smaller and contain fewer disulfide bonds, and thus more efficient for SHuffle bacteria to produce. In particular, IK-17 and MCPC603 are single chain variable fragments (scFvs), so they are only comprised of a variable light and variable heavy region. Upon advice from Dr. Leitinger (see Modeling and Attributions for more details), we decided to immobilize both these different antibodies on the test line so we'd have a higher chance of successful and specific binding to oxLDL.
                        </p>
                    </div>

                    <div class="col-md">
                        <img class="abimg" src="https://static.igem.wiki/teams/4477/wiki/scfv.svg" />
                    </div>
                </div>
            </div>
            <div class="container">
                <div class="body-section">
                    <img class="abimg" src="https://static.igem.wiki/teams/4477/wiki/vhh.svg"/>
                    <div class="text-wrapper"> 
                        <h4>Anti-IgG Design</h4>
                        <p>
                            We designed a secondary antibody to be immobilized on the control line. This antibody was a nanobody (VHH), a variable region heavy chain only antibody naturally found in camelids. The sequence we used was encoded for an anti-mouse VHH that binds to the constant domain on mouse IgG1 antibodies. A toolbox of anti–mouse and anti–rabbit IgG secondary nanobodies | Journal of Cell Biology | Rockefeller University Press (rupress.org) This allowed the VHH to bind to the hybrid full-length primary antibody because it recognized the mouse constant domain within it.
                        </p>
                    </div>
                </div> 
            </div>

            <div class="container">
                <div class="body-section">
                    <div class="text-wrapper">
                        <h2>Biosafety</h2>
                        <p>As synthetic biologists, it is our responsibility to take in all considerations in terms of the safety of our device. We met with experts in the field of biosafety to implement their expertise into our device design.
                            The first expert we met to discuss lab safety and biocontainment measures was Jennifer Kershner, a biosafety officer at the University of Virginia. We discussed safety practices to employ in the lab to ensure that not only we stay safe, but that we also prevent release of biological agents in the environment. She mentioned that because we are only working with bacteria in the lab and not including them in our device that will leave the lab, we do not need to employ a kill switch of any sorts in our project design. In terms of the safety protocols we should instruct our consumers to use, we discussed the use of an alcohol wipe before the consumer pricks their finger on our device and then a bandaid for the consumer to put on afterwards to prevent the wound from getting infected. We have decided to put these two in the packaging of the test strip. Additionally, we asked about what measures a consumer should take when disposing of our test due to the use of blood. She explained that because the test will only require a small amount of blood and as long as the recombinant antibodies are neutralized, the consumer should be able to dispose of the test in a regular trash can. 
                            This meeting gave us an idea about the considerations we need to make when designing the device to ensure the easiest process for our consumers so that they can stay safe. 
                        </p>
                    </div>
                </div>
            </div>

            <div class="container">
                <div class="body-section">
                    <div class="text-wrapper">
                        <h2>Hardware</h2>
                        <p>The hardware for our device consists of a 3D-printed plastic cassette we designed that encapsulates the LFA test strip. A lancet for finger-pricking is also attached to the cassette. The cassette allows the LFA to be held in place and users to handle the device safely. When designing the cassette, we made sure it was both safe and environmentally-friendly, taking both the consumers' health and waste disposal processes into account. Our 3D model prototype also allows for mass production of our product easily.</p>
                    </div>
                </div>
            </div>
        </div>
        <div class="spacer-div"></div>
    </div>
</div>

<div class="footer-preface curvy-divider">
    <div class="footer-divider">
        <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
            <path d="M985.66,92.83C906.67,72,823.78,31,743.84,14.19c-82.26-17.34-168.06-16.33-250.45.39-57.84,11.73-114,31.07-172,41.86A600.21,600.21,0,0,1,0,27.35V120H1200V95.8C1132.19,118.92,1055.71,111.31,985.66,92.83Z" class="shape-fill"></path>
            </svg>
    </div>
</div>
    








{% endblock %}


