{% extends "layout.html" %}
  
{% block title %}Engineering Success{% endblock %}
{% block lead %}Demonstrate engineering success in a part of your project by going through at least one iteration of the engineering design cycle.{% endblock %}

{% block page_content %}
<div class="standard-header curvy-divider banner-v2">
  <h1> Engineering Success </h1>
  <div class="wave2">
    <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
        <path d="M321.39,56.44c58-10.79,114.16-30.13,172-41.86,82.39-16.72,168.19-17.73,250.45-.39C823.78,31,906.67,72,985.66,92.83c70.05,18.48,146.53,26.09,214.34,3V0H0V27.35A600.21,600.21,0,0,0,321.39,56.44Z" class="shape-fill"></path>
    </svg>
  </div>
</div>

<div class="d-flex flex-column justify-content-center align-items-center curvy-divider">
  <div class="body-container wave-div narrow">
      <div class="container">
        <div class="about-section d-flex flex-row justify-content-center align-items-center">
          <div class="wrapper">
            <h2>The Engineering Design Cycle</h2>
            <p>Throughout the course of our project, we faced many challenges and setbacks we needed to overcome. We also talked to stakeholders and researchers to gain knowledge we otherwise would not have obtained. These warranted repeated changes to our design and tweaks to aspects of our project. Below, we show our iterations through the engineering design cycle, both due to integrated human practices and experimental discoveries. </p>
          </div>
          <div class="img-wrapper">
            <img style="padding: 2rem;"src="https://static.igem.wiki/teams/4477/wiki/images/eng-des-cycl.svg" alt="lfa bunch">
          </div>
        </div>
      </div>
  </div>
  <div class="gray-divider">
    <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
        <path d="M985.66,92.83C906.67,72,823.78,31,743.84,14.19c-82.26-17.34-168.06-16.33-250.45.39-57.84,11.73-114,31.07-172,41.86A600.21,600.21,0,0,1,0,27.35V120H1200V95.8C1132.19,118.92,1055.71,111.31,985.66,92.83Z" class="shape-fill"></path>
    </svg>
  </div>
</div>

<div class="gray d-flex flex-row justify-content-center body-padding">
  <div class="spacer-div"></div>

  <div class="body-container d-flex flex-column align-items-center justify-content-center" data-aos="fade-up" data-aos-duration="1000" data-aos-easing="ease-out-sine">
    <div class="container">
      <div class="body-section top-page">
        <div class="text-wrapper">
          <div class="engr-cycle">
            <h2>Cycle 1</h2>
            <div class="cyc">
              <h4>Design</h4>
              <p>To design the four antibodies required for our project, we <span class="bold-text">BrioBricked</span> four inserts as a composite part. Each composite part has a strong T7 inducible promoter, enabling IPTG induction, an <span class="italic-text">E. coli</span> codon-optimized ribosomal binding site (RBS), the antibody coding sequence, and a terminator. The RBS and antibody coding sequence were designed to serve a different purpose for each antibody. Legal restriction enzymes were also placed between each subcomponent. Next, we decided to use <span class="bold-text">pSB3K3</span> for our vector, a low-copy number plasmid, which is important to reduce the metabolic burden on our cells.<span class="tt" data-bs-toggle="tooltip" data-bs-html="true" data-bs-placement="top" title="<p class='italic-text'>Karim, A. S., Curran, K. A. & Alper, H. S. Characterization of plasmid burden and copy number in Saccharomyces cerevisiae for optimization of metabolic engineering applications. FEMS Yeast Res. 13, 107–116 (2013).</p>"><sup>1</sup></span></p>
            </div>

            <div class="cyc">
              <h4>Build</h4>
              <p>We used traditional cloning (TC) to incorporate our insert into the pSB3K3 vector in <span class="italic-text">E. coli</span> DH5-alpha cells, however, we received low transformation efficiency.  We modified our TC by mixing the digested insert and vector directly in a tube for ligation without performing gel electrophoresis. Recognizing the possibility of our original mRFP gene fragment being re-ligated back into the vector after being digested with restriction enzymes, we screened transformants based on the colonies’ color. Colonies with our insert will appear white, while those with the re-ligated mRFP gene will appear red.  After cloning our antibody sequences into DH5-alpha cells and selecting only for white colonies, we utilized <span class="bold-text">restriction mapping and sequencing</span> to verify that the DH5-alpha cells contained our antibody inserts. We then transformed <span class="italic-text">E. coli</span> SHuffle and utilized restriction mapping and sequencing to verify SHuffle contained the correct insert.</p>
            </div>

            <div class="cyc">
              <h4>Test</h4>
              <p>To test if our devices expressed our antibodies of interest, we <span class="bold-text">induced SHuffle cells</span> to produce protein and then conducted SDS-PAGE to verify if protein bands were present at the correct molecular weight. However, we observed low expression. Based on this, we changed various parameters of protein induction (incubation time, IPTG concentration, and temperature), but we still did not see significant expression of our target antibodies.</p>
            </div>

            <div class="cyc">
              <h4>Learn</h4>
              <p>Based on our testing results, review of academic articles, and expert opinions, we learned that other vectors can allow for <span class="bold-text">higher protein expression</span>.  Since our pSB3K3 vector is a low copy number plasmid, it may have resulted in too few transcripts being created despite induction.  </p>
            </div>
          </div>
        </div>
      </div>
      <div class="body-section">
        <div class="text-wrapper">
          <div class="engr-cycle">
            <h2>Cycle 2</h2>
            <div class="cyc">
              <h4>Design</h4>
              <p>We are currently re-designing our constructs so that they use the <span class="bold-text">pET-21b(+) expression vector</span> used by Robinson et al., a high copy number plasmid designed for use in T7 induction.</p>
            </div>
          </div>
        </div>
      </div>

      <div class="body-section">
        <div class="text-wrapper">
          <h2>Future Iterations</h2>
            <p>If modifying these variables does not increase protein yields, we will redesign our BioBrick construct. We will design our RBS to have maximum translational efficiency rather than 75% efficiency and use Robinson promoters to <span class="bold-text">increase protein expression</span>. </p>
        </div>
      </div>

      <div class="body-section">
        <div class="text-wrapper">
          <h2>Conclusion</h2>
          <p>Following the Engineering Design Cycle, we were successfully able to build our SHuffle cells and test protein induction.</p>
        </div>
      </div>
    </div>
  </div>
    <!-- SPACING -->
  <div class="spacer-div"></div>
</div>

<div class="footer-preface curvy-divider">
  <div class="footer-divider">
      <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
          <path d="M985.66,92.83C906.67,72,823.78,31,743.84,14.19c-82.26-17.34-168.06-16.33-250.45.39-57.84,11.73-114,31.07-172,41.86A600.21,600.21,0,0,1,0,27.35V120H1200V95.8C1132.19,118.92,1055.71,111.31,985.66,92.83Z" class="shape-fill"></path>
          </svg>
  </div>
</div>

{% endblock %}
