{% extends "layout.html" %}
  
{% block title %}Experiments{% endblock %}
{% block lead %}Describe the research, experiments, and protocols you used in your iGEM project.{% endblock %}

{% block page_content %}
<div class="standard-header banner-v2 curvy-divider">
  <h1> Experiments </h1>
  <div class="wave2">
    <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
        <path d="M321.39,56.44c58-10.79,114.16-30.13,172-41.86,82.39-16.72,168.19-17.73,250.45-.39C823.78,31,906.67,72,985.66,92.83c70.05,18.48,146.53,26.09,214.34,3V0H0V27.35A600.21,600.21,0,0,0,321.39,56.44Z" class="shape-fill"></path>
    </svg>
  </div>
</div>

<div class="d-flex flex-column justify-content-center align-items-center">
  <div class="body-container wave-div narrow">
      <div class="container">
        <div class="about-section d-flex flex-row justify-content-center align-items-center">
          <div class="wrapper">
            <h2>Introduction</h2>
            <p>We performed various experiments during the course of our project. In this section, we will give an overview of the experiments we ran and supplementary data. Protocols are described in detail on the Notebook page under Wet Lab. Our Detailed Workflow document containing all of our protocols can be found here: [add button for detailed workflow]</p>
          </div>
          <div class="img-wrapper bunch-of-lfas">
            <img style="max-height: 15rem; padding: 2rem;"src="https://static.igem.wiki/teams/4477/wiki/images/lfa-group.svg" alt="lfa bunch">
          </div>
        </div>
      </div>
  </div>
</div>

<div class="d-flex flex-row justify-content-center body-padding">
  <div class="index-padding">
    <div class="fixed-index">
      <p>Index</p>
    </div>
  </div>

  <div class="body-container d-flex flex-column align-items-center justify-content-center">
  
    <div class="container">
      <div class="body-section top-page">
        <div class="text-wrapper">
          <h2>Plasmid Insertion</h2>
          <p>
            Once the ligation product was obtained, we transformed the E. coli chassis with the heat shock method. The temperature imbalance induced by this method disrupts the lipid bilayer membrane of E. coli, by making bigger pores on its surface; this allows the plasmid to enter the cell. (Plasmid Insertion Reference)
            The method was implemented by heating the cells which allowed the DNA to flow through. After the DNA is in the cells are then cooled on ice to close up the pores fully allowing the insertion of the plasmid. This insured that the DNA we were targeting would be in the cells, and we could targetize it later.
            </p>
        </div>
      </div>
    </div>
  
    <div class="container">
      <div class="body-section">
        <div class="text-wrapper">
          <h2>Verifying Plasmids</h2>
          <p>
            Many procedures go into assembling the final functional plasmid. Unfortunately, that introduces many opportunities for error. Thus checkpoints are integrated into our plasmid assembly workflow. 
            (Plasmid Verification Reference.)
            As a way to mitigate this, Gel electrophoresis analysis is run with each step after we introduce changes to DNA. This requires us to run gels with our DNA so we can check the corresponding band sizes sequences. Once the assembled plasmid is obtained in addition to gel electrophoresis the DNA is sent out for sequencing. Lastly, the functional test which is acquiring the protein products from our chassis verifies our plasmid.
            </p>
        </div>
      </div>
    </div>
    
    <div class="container">
      <div class="body-section">
        <div class="text-wrapper">
          <h2>Verifying Restriction Enzyme Function</h2>
          <p>Restriction enzymes are delicate proteins that must be kept at cold temperature consistently otherwise they will denature and lose all function. Handling them was challenging so we developed a method to test their functionality whenever they were used. For example if weâ€™re intending to cut a vector out of a plasmid using two restriction enzymes, four digest protocols are performed. Verifying restriction enzyme function Reference. The four digests were a double cut, single cut using the first enzyme, single cut using the second enzyme, and an uncut. Once digest products are obtained they are run on an analysis gel to compare and contrast. If the results confirm our predictions of the band lengths and number then our digest is successful. Thus the enzymes are verified and vice versa. We could then proceed with further experiments after this experiments were run, otherwise trouble shooting would be needed.</p>
        </div>
      </div>
    </div>

    <div class="container">
      <div class="body-section">
        <div class="text-wrapper">
          <h2>Plasmid Sequencing</h2>
          <p>In order for this process to occur, it required us to send in our primers that we had designed in order to make sure the inserts got cloned into the backbones. Plasmid Sequencing References. Our designs were under the constrains that our plasmids sequences were ~20 bp in length and ~50% GC content. With one primer to start the transcription process  ~1 kb downstream of the primer will get sequenced. Place primer accordingly (can be ~100 or ~200 bp upstream of region of interest and your sequenced DNA will still contain the region of interest). We would then drop primer and DNA samples in the respective drop box (and results will come back in about 2 weeks!</p>
        </div>
      </div>
    </div>

    <div class="container">
      <div class="body-section">
        <div class="text-wrapper">
          <h2>Transformations</h2>
          <p>Once transformations are done, the transformed cells are plated on LB and an antibiotic of choice plates. Our plasmids are designed to contain a resistance vector to a specific antibiotic, such as Chloramphenicol. Transformation References
            This enables only transformed and functional cells to survive on the plates thus obtaining the desired synthetic device. Negative and positive controls are transformed and plated alongside our device to verify the effectiveness of the antibiotic LB plates thereby reducing error.
            </p>
        </div>
      </div>
    </div>

    <div class="container">
      <div class="body-section">
        <div class="text-wrapper">
          <h2>Protein Induction</h2>
          <p>This process typically occurs after inoculation of the cells. After the incubation of the cell cultures, the proteins are induced with IPTG, which stimulates the process of transcription. A Deep Dive Into Induction With IPTG | GoldBio, n.d.)
            They are then incubated overnight in order to proliferate transcription and transduction of the sequence we tend to isolate. The ability to induce this was based on our protein sequence design via the vector that was registered through iGEM.
            </p>
        </div>
      </div>
    </div>

    <div class="container">
      <div class="body-section">
        <div class="text-wrapper">
          <h2>SDS-PAGE</h2>
          <p>
            After our proteins have been isolated, we ran an SDS Page Gel to separate them on the order of molecular weight. This typically consisted of a ladder at the forefront of the gel and lysed cell protein samples. SDS PAGE Reference
            The gels were run in a range of 80-140 V depending on the molecular buildup of the protein. The proteins were compared with the ladder to determine how well the ladder ran and to see if the molecular weight of the segments lined up with what was anticipated. If the molecular weight matched, we can then transfer them using western blot techniques.
            </p>
        </div>
      </div>
    </div>
  </div>
<!-- SPACING -->
    <div class="spacer-div"></div>
</div>




{% endblock %}
